docs/sources/helm-charts/mimir-distributed/configure/configure-redis-cache.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Configure Redis cache"
+menuTitle: "Redis cache"
+description: "Learn how to configure Grafana Mimir to use external Redis as cache"
+---
+
+# Configure Redis cache
+
+Beside built-in support for Memcached, Mimir also supports external Redis for chunks-cache, index-cache, results-cache and metadata-cache. To use Redis, we must disable the build-in Memcached
+from `mimir-distributed` helm chart by making sure the cache configuration is not enabled. We can explicitly set `*-cache.enabled` to false or remove the whole `*-cache`
+block from values file so that default setting disables Memcached.
```

Comment:
There's two options here (explicitly setting `enabled` to `false` or removing the blocks altogether) - is there one we recommend?